test/runtests.jl

```diff
@@ -1,9 +1,7 @@
-import GLFW
-
 travis = get(ENV, "TRAVIS", "") == "true"
 
 try
-	GLFW.Init()
+	eval(Main, Expr(:import, :GLFW))
```

Comment:
Yep, the `try...catch` introduces a new scope so you need to force evaluation in the `Main` module.